src/server/scripts/Outland/zone_netherstorm.cpp

```diff
@@ -668,7 +668,12 @@ class npc_maxx_a_million_escort : public CreatureScript
 enum CaptainTyralius
 {
     NPC_CAPTAIN_TYRALIUS    = 20787,
+    NPC_ETHEREUM_PRISONER   = 20825,
+    SPELL_GREEN_BANISH      = 32567,
```

Comment:
not used?

Reply:
Right, it's from a previous fix attempt. Forgot to remove that, thanks!